Guides/Getting Started.md

```diff
@@ -8,23 +8,29 @@ After installing `IGListKit`, creating a new list is easy.
 
 ### Creating a section controller
 
-Creating a new section controller is simple. You subclass `IGListSectionController` and conform to the `IGListSectionType` protocol. Once you conform to `IGListSectionType`, the compiler will make sure you implement all of the required methods.
+Creating a new section controller is simple. All you have to do is subclass `IGListSectionController` and override at least `cellForItemAtIndex:` and `sizeForItemAtIndex:`.
```

Comment:
Not 100% sure I agree with that Jesse, whilst the majority will override `didUpdateToObject:` in the case of decoration views such as adverts or loading views that don't have an object necessarily bound to them I don't see the reason to have the method overridden